src/plugins/lua/known_senders.lua

```diff
@@ -230,6 +360,20 @@ if opts then
       augmentations = { string.format("timeout=%f", redis_params.timeout or 0.0) }
     })
 
+    rspamd_config:register_symbol({
+      name = settings.symbol_check_mail_local,
+      type = 'normal',
+      callback = verify_local_replies_set,
+      score = 1.0
+    })
+
+    rspamd_config:register_symbol({
+      name = settings.symbol_check_mail_global,
+      type = 'normal',
+      callback = check_known_incoming_mail_callback,
+      score = 1.0
```

Comment:
We should rather have negative scores for these symbols I suppose

Reply:
We can set it in the config later maybe?

Reply:
Which negative scores should we have?